### ISSUE TITLE
Change last sentence in About Section

### ISSUE THREAD

**Issue by stratofax** [stratofax/cbailylaw#49]
Edit the About section as follows (the **boldface** should not be included on the webpage, but is added here to highlight the changes)

Christine Baily is licensed to practice in Massachusetts **and** Connecticut. **She is seeking admission in Maine**.

**Comment by ratulkhan74** [comment]
I didn't get the change in about section  can you please explain again

**Comment by stratofax** [comment]
Yes, happy to clarify. Here's the current text of the last paragraph (`p` tag) in the About section:

> Christine Baily is licensed to practice in Massachusetts, Connecticut, and Maine (pending admission).

Please change that paragraph to read as follows:

```
Christine Baily is licensed to practice in Massachusetts and Connecticut. She is seeking admission in Maine.
```

**Comment by ratulkhan74** [comment]
changes done

**Comment by stratofax** [comment]
Thanks and well done!

**Comment by stratofax** [comment]
Closed with PR #49
